clippy.toml

```diff
@@ -1 +0,0 @@
-msrv = "1.70.0"
```

Comment:
Just confirming: clippy will look at `workspace.package.rust-version` instead?  That's great.

Reply:
It seems to, if there is a `rust-version.workspace = true` in the `Cargo.toml` of the member package.

https://doc.rust-lang.org/cargo/reference/workspaces.html#the-package-table

Reply:
It seems to:

> [msrv](https://doc.rust-lang.org/clippy/lint_configuration.html#msrv)
> The minimum rust version that the project supports. Defaults to the rust-version field in Cargo.toml

https://doc.rust-lang.org/clippy/lint_configuration.html#msrv